framework/include/base/Registry.h

```diff
@@ -222,6 +222,11 @@ class Registry
     return getRegistry()._data_file_paths;
   }
 
+  ///@{ Don't allow creation through copy consturction or assignment
+  Registry(Registry const &) = delete;
+  void operator=(Registry const &) = delete;
```

Comment:
Do you want to do the same for move construction/assignment?

Reply:
Yes, but did you notice that the signature was wrong?! It was wrong in the `Registry` class. Fixed everywhere!

Reply:
I did not until I saw your newest commit 😄